src/components/auth/SignInScreen.js

```diff
@@ -66,20 +70,17 @@ export default class SignInScreen extends React.Component {
   render() {
     return (
       <SafeAreaView style={styles.container}>
-        <KeyboardAvoidingView style={styles.container} behavior='padding' enabled>
           <TouchableWithoutFeedback
             style={styles.container}
             onPress={Keyboard.dismiss}>
-            <View style={styles.container}>
               <Container style={styles.infoContainer}>
-                <View style={styles.container}>
+                <View style={styles.container1}>
                   {/* Email */}
                   <Item style={styles.itemStyle}>
-                    <Ionicons style={styles.iconStyle} name="ios-mail" />
                     <Input
                       style={styles.input}
-                      placeholder={i18n.t('Email')}
-                      placeholderTextColor={COLORS.lightblue}
+                      placeholder={"E-mail"}
```

Comment:
## Translation package does not work with hyphenated or words with spaces. 
### Instead, use {i18.t('Email')} for your placeholder.

Reply:
Translation will be a whole other REF. For now, feel free to use and abuse your string constants